railties/lib/rails/generators/rails/plugin/templates/gitignore

```diff
@@ -2,8 +2,10 @@
 log/*.log
 pkg/
 <% if with_dummy_app? -%>
+<% if sqlite3? -%>
```

Comment:
:+1: As a result, can we move tests about sqlite3 to `SharedGeneratorTests` too?
https://github.com/bogdanvlviv/rails/blob/db4cbf27a6ced61b83e68ef1187a9ac67b8ae5bb/railties/test/generators/app_generator_test.rb#L678...L692